content/authors/moses-mwangi/index.md

```diff
@@ -0,0 +1 @@
+Moses Mwangi is a self-taught and solution-driven Java developer. He enjoys teaming up with all levels of developers to solve complex problems and learning from each other.
```

Comment:
Hey Moses. Edit your profile like other people. Open content/authors folder and copy the format. Thanks